src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -725,14 +725,23 @@ static int enumerate_partitions(dev_t devnum) {
 }
 
 static int add_mounts(void) {
+        const char *euclean_path;
         dev_t devno;
         int r;
 
         r = get_block_device_harder("/", &devno);
+        if (r == -EUCLEAN) {
+                euclean_path = "root file system";
+                goto euclean;
+        }
```

Comment:
I think it'd be nicer to add an inline helper function that takes this label as an argument and wraps `log_error_errno()` with the appropriate message. This way the message itself will not have to be defined in a thousand places and goto will not need to be used.

Reply:
makes sense